src/styles/blocks/button.scss

```
@@ -0,0 +1,77 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  width: 100%;
+  margin-bottom: 30px;
+  @include onTablet {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  background-color: #cd4d31;
+
+  &:hover {
+    background-color: #af3419;
+  }
+  &:active {
+    background-color: #ea340d;
+  }
+  &:disabled {
+    background-color: #dd836f;
+  }
+  &__link {
```

add page brakes like this:
```suggestion
  &:hover {
    background-color: #af3419;
  }
  
  &:active {
    background-color: #ea340d;
  }
  
  &:disabled {
    background-color: #dd836f;
  }
  
  &__link {
```